tests/integration/targets/mso_schema_template_contract_service_graph/tasks/main.yml

```diff
@@ -37,12 +32,14 @@
     <<: *mso_info
     schema: '{{ mso_schema | default("ansible_test") }}_2'
     state: absent
+  when: version.current.version is version('3.3', '>=')
```

Comment:
when most of test cases are using the condition when: version.current.version is version('3.3', '>=')

we can replace it by using block (block:)

Reply:
ok makes sense, thanks! changed it